screens/AboutScreen.js

```diff
@@ -1,59 +1,88 @@
 import { ScrollView, Text } from "react-native";
 import { Avatar, Card, ListItem } from "react-native-elements";
-import { PARTNERS } from "../shared/partners";
-import { useState } from "react";
+//import { PARTNERS } from "../shared/partners";
+//import { useState } from "react";
```

Comment:
go ahead and remove these commented out lines.  If you need them later on, git has a record of them.